.github/workflows/sitl_tests.yml

```diff
@@ -96,7 +96,7 @@ jobs:
         PX4_HOME_LON: ${{matrix.config.longitude}}
         PX4_HOME_ALT: ${{matrix.config.altitude}}
         PX4_CMAKE_BUILD_TYPE: ${{matrix.config.build_type}}
-      run: test/mavsdk_tests/mavsdk_test_runner.py --speed-factor 20 --abort-early --model ${{matrix.config.model}} --upload test/mavsdk_tests/configs/sitl.json --verbose
+      run: test/mavsdk_tests/mavsdk_test_runner.py --speed-factor 3 --abort-early --model ${{matrix.config.model}} --upload test/mavsdk_tests/configs/sitl.json --verbose
```

Comment:
Doesn't it also work with speed factor 20 now?

Reply:
Let's see, I've removed this commit for now.

Reply:
seems to be okay as well without it